docs/en/observability/synthetics-manage-monitors.asciidoc

```diff
@@ -0,0 +1,17 @@
+[[synthetics-manage-monitors]]
+= Manage synthetic monitors
```

Comment:
I will wait to build this out until we've finalized other content since this is something that doesn't exist in the current docs yet anyway.